src/elements/ProgressBar/ProgressBar.js

```diff
@@ -7,34 +7,36 @@ import PropTypes from 'prop-types';
 
 export default class ProgressBar extends React.Component {
 	static propTypes = {
-		onComplete: PropTypes.func,		// A function to call when the loader finishes
-		increments: PropTypes.number,	// The number of increments you want to break the loader into
-		autoStart: PropTypes.bool,		// Start the animation immediately after initialization
-		duration: PropTypes.number,		// Time in milliseconds the loader should run for
-		delay: PropTypes.number,		// Delay before auto-starting the animation
-		completeDelay: PropTypes.number	// Delay in milliseconds before onComplete fires when the loader is complete
+		onComplete: PropTypes.func,				// A function to call when the loader finishes
+		increments: PropTypes.number,			// The number of increments you want to break the loader into
+		autoStart: PropTypes.bool,				// Start the animation immediately after initialization
+		duration: PropTypes.number.isRequired,	// Time in milliseconds the loader should run for
+		delay: PropTypes.number,				// Delay before auto-starting the animation
+		completeDelay: PropTypes.number,		// Delay in milliseconds before onComplete fires when the loader is complete
+		height: PropTypes.number				// Height of progress bar in px
```

Comment:
I forgot to mention that this prop should allow a number or a string so it can be passed both ways:
```
<ProgressBar height="45" />
....
<ProgressBar height={calculateHeight(x) /* returns number */} />
```

When the value gets applied, you would just explicitly coerce it to a number via `Number(this.props.height)`